src/OSPSuite.Core/Domain/ParameterExtensions.cs

```diff
@@ -46,5 +46,18 @@ public static bool IsConstantParameter(this IParameter parameter)
       {
          return parameter.IsFixedValue || parameter.Formula.IsConstant() || parameter.Formula.IsDistributed();
       }
+
+      public static bool IsGlobalExpression(this IParameter parameter)
```

Comment:
do we agree that those could be changed to IWIthName and then you do not need the method in the Object Above?